src/compilers/for_expression.cr

```diff
@@ -22,18 +38,24 @@ module Mint
           JS
         end
 
+      index =
+        if index_arg
+          "const #{js.variable_of(index_arg.not_nil!)} = _i"
```

Comment:
```suggestion
          "const #{js.variable_of(index_arg)} = _i"
```

Reply:
I thought there is no overload for `variable_of(variable : Nil)`

Reply:
`index_arg` can't be `Nil` at this point.

Reply:
Yep I figured :facepalm: